Sources/SnapshotTesting/Extensions/UInt8+diff.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+extension UInt8 {
+    func diff(between other: UInt8) -> UInt8 {
```

Comment:
It'd be nice to have a little bit of doc describing the intent behind this method 🙏🏽

Reply:
What do you have in mind?

I'm generally pretty keen to describe the intent of a given function although this seems like something that is simple enough it can be used independently from the intent of its original purpose.

Excuse the somewhat exaggerated example as I can't think of a better one right now but this is an example of where I wouldn't want to describe the original intent of a function because of its mathematical/general purpose nature.
```swift
  /// This is used to calculate the hypotenuse of a triangle
  func sqrt(number: Double)
```